controller/src/main/java/io/pravega/controller/server/rpc/grpc/GRPCServer.java

```diff
@@ -72,8 +76,14 @@ public GRPCServer(ControllerService controllerService, GRPCServerConfig serverCo
         }
 
         if (serverConfig.isTlsEnabled() && !Strings.isNullOrEmpty(serverConfig.getTlsCertFile())) {
-            builder = builder.useTransportSecurity(new File(serverConfig.getTlsCertFile()),
-                    new File(serverConfig.getTlsKeyFile()));
+            try {
+                builder = builder.useTransportSecurity(new File(serverConfig.getTlsCertFile()),
+                        new File(serverConfig.getTlsKeyFile()));
+                SslContext ctx = GrpcSslContexts.forServer(new File(serverConfig.getTlsCertFile()), new File(serverConfig.getTlsKeyFile())).protocols(serverConfig.getTlsProtocolVersion().split(",")).build();
```

Comment:
Please ensure the logic related to config like getTlsProtocolVersion().split(",") is perfomed in GRPCServerConfig class.

Reply:
Moved away the split or regex logic from here.